presto-main/src/main/java/com/facebook/presto/sql/analyzer/Scope.java

```diff
@@ -185,13 +194,28 @@ private static boolean isColumnReference(QualifiedName name, RelationType relati
         return Optional.empty();
     }
 
+    @Override
+    public String toString()
+    {
+        return toStringHelper(this)
+                .addValue(relationId)
```

Comment:
Other fields (parent, relationType) are surely more important, but IMO too verbose to squeeze them into useful toString.  Still, toString with relationId (which in turn includes node type) is much more useful than no toString at all